lib/iris/tests/unit/analysis/maths/test_exponentiate.py

```diff
@@ -0,0 +1,94 @@
+# (C) British Crown Copyright 2015, Met Office
```

Comment:
That is far from being our only problem I suspect... Is it really complaining about the license header??